dev.yaml

```diff
@@ -0,0 +1,23 @@
+services:
+  dirsrv:
+    build:
+      context: .
+      dockerfile: Containerfile-dev
+      args:
+        CUSTOM_CRT_URL: ${CUSTOM_CRT_URL}
+    hostname: dirsrv
+    container_name: dirsrv
+    ports:
+      - "3389:3389"
+      - "3636:3636"
+    init: true
+    entrypoint: ["sleep", "infinity"]
+    cap_add:
+      - SYS_PTRACE
+    env_file:
+      - path: ./dev-defaults.env
```

Comment:
Please move these env files under `.devcontainer`, and omit `dev-`. I think it's clear that this is a devcontainer, no need to put Dev/Development everywhere.

Reply:
Done.